### ISSUE TITLE
SP_TRITH always have unknown population modifier

### ISSUE THREAD

**Issue by aschuch247** [freeorion/freeorion#2777]
### Environment

* **FreeOrion Version**: 3ac143d
* **Fetched as**
  * Compiled from source

### Description

Enemy colonies of `SP_TRITH` always show unknown population modifier.

### Expected Result

The reason of the population minus shall be shown.

### Steps to reproduce

* Start a new game, using seed `9` and only a single AI, also chose `SP_TRITH` as species.
* The own capital planet has a population of 32 (large radiated).
* Build super testers.
* The enemy capital has a target population of 27.4 (large radiated).
* Furthermore, all enemy colonies of `SP_TRITH` show a negative unknown population modifier.

**Comment by geoffthemedio** [comment]
Does it only happen after https://github.com/freeorion/freeorion/pull/2774 ?

**Comment by aschuch247** [comment]
@geoffthemedio: No. I just compiled be647f6c3c6c43845525242a8fd019ae2319e225 and the same happens.

**Comment by aschuch247** [comment]
![trith](https://user-images.githubusercontent.com/13714468/75123003-8fc20c00-569b-11ea-8f6a-82291cfc4388.png)

A Derthrean colony exists in the same system as the Trith capital. My guess is that the xenophobic frenzy is just misplaced in the client.
